docs/csharp/language-reference/keywords/reference-tables-for-types.md

```diff
@@ -13,25 +13,20 @@ ms.assetid: 1d2b6a9b-31fd-4fa6-a7b2-bfd9c806ac8a
 
 The following reference tables summarize the C# types:
 
-- [Built-in Types Table](built-in-types-table.md)
-
-- [Integral types](integral-types-table.md)
-
-- [Floating-point types](floating-point-types-table.md)
-
-- [Default values](default-values-table.md)
-
-- [Value types](value-types-table.md)
-
-- [Implicit numeric conversions](implicit-numeric-conversions-table.md)
-
-- [Explicit Numeric Conversions Table](explicit-numeric-conversions-table.md)
-
-For information on formatting the output of numeric types, see [Formatting Numeric Results Table](formatting-numeric-results-table.md).
+|Table|Description|
+|---------|---------|
+|[Built-in types](built-in-types-table.md)|Lists built-in C# types and corresponding .NET types.|
+|[Integral types](integral-types-table.md)|Provides an overview of the integral types.|
+|[Floating-point types](floating-point-types-table.md)|Provides an overview of the floating-point types.|
+|[Value types](value-types-table.md)|Lists C# value types.|
```

Comment:
in might be good to add "built-in" here

Reply:
@rpetrusha the table in question contains entries for `enum` and `struct` (stating it's a user-defined structure) types. So, that table covers not only concrete types (like `bool` and numerical ones), but all possible enumerations and structures. If we add "built-in" here. what would be excluded then?

Reply:
@pkulikov, it seems that I attached my comment to the line I didn't want to attach it to, and didn't attach it to the two lines that I did want to attach it to. Even given that, though, my suggestion seems to cloud rather than clarify the issue. I'll merge your PR now.